logaccess/loki/src/main/kotlin/LokiLogFileProvider.kt

```diff
@@ -104,7 +104,18 @@ class LokiLogFileProvider(
                 }
 
                 if (statements.size >= config.limit && deDuplicatedStatements.isNotEmpty()) {
-                    downloadChunk(statements.last().timestamp, deDuplicatedStatements)
+                    val lastTimestamp = statements.last().timestamp
+
+                    if (lastTimestamp != from) {
+                        downloadChunk(lastTimestamp, deDuplicatedStatements)
+                    } else {
+                        logger.warn(
```

Comment:
Should we make this an error because it is much more likely that log lines got lost than that the lines with the same timestamp match the chunk size exactly?

Reply:
Yes, I now made it an error in the log.